packages/next-dashboard/src/components/Statistic.js

```diff
@@ -15,6 +15,7 @@ export type Props = {
   description?: R.Node,
   status?: Statuses,
   direction?: 'up' | 'down',
+  positive?: 'up' | 'down' | 'n/a',
```

Comment:
```suggestion
  positive?: 'up' | 'down' | null,
```

Reply:
Look I'm tired and have been all day, I literally have no idea what I was thinking here.

Reply:
Does it make sense to have a `null` option here? Do we have a neutral type of up/down arrow? (like, grey or blue or something?)

Reply:
Yes, I added a grey arrow in the pier project.

Reply:
Right, my bad, didn't see that! Yeah, this is good then :)